### ISSUE TITLE
Shopify Theme Serve from Localhost SSL Errors

### ISSUE THREAD

**Issue by ckizer** [Shopify/shopify-cli#1470]
Shopify Theme Serve says:
```
Please open this URL in your browser:
┃ http://127.0.0.1:9292
```

Don't so results in a page that mostly refuses to load due to SSL errors. Any suggestions? Obviously I can load the remote URL, but if local URL won't work with dawn theme, maybe it shouldn't be suggested? Or it might be something obvious I'm overlooking.

Screenshot.

<img width="1234" alt="ScreenShot 2021-08-25 at 01 54 34@2x" src="https://user-images.githubusercontent.com/191738/130760588-d45474e0-0f10-42b7-a424-a4a8302d8dc0.png">

**Comment by dleatherman** [comment]
I've found you can use the theme preview URL for Safari instead of the local. This isn't a solution, but a short term fix.

**Comment by dhruveonmars** [comment]
Yeah, remote url is a temporary fix if working with Safari, but not great as it doesn't have the auto-refresh that the local version has.
Hopefully see a fix soon

**Comment by macournoyer** [comment]
`shopify theme serve` is currently only available in Chrome:

https://shopify.dev/themes/getting-started/create#step-4-preview-your-theme

For other browsers, you can use the remote URL as you've said.

https://github.com/Shopify/shopify-cli/pull/1408 should add support for serving the preview over SSL and allow it to be used in Safari
